_posts/2016-03-21-windows-support.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Windows support update
+categories: tech-issues future-plans
```

Comment:
I used these categories as they are the same as we used to java webstart support. I can add deployment if you like.